common/src/main/java/com/alibaba/nacos/common/notify/NotifyCenter.java

```diff
@@ -250,13 +250,12 @@ private static void addSubscriber(final Subscriber consumer, Class<? extends Eve
     private static boolean removeSubscriber(final Subscriber consumer, Class<? extends Event> subscribeType) {
         
         final String topic = ClassUtils.getCanonicalName(subscribeType);
-        if (INSTANCE.publisherMap.containsKey(topic)) {
-            EventPublisher publisher = INSTANCE.publisherMap.get(topic);
-            publisher.removeSubscriber(consumer);
-            return true;
+        EventPublisher eventPublisher = INSTANCE.publisherMap.get(topic);
```

Comment:
I think the old implementation is better for readablilty

Reply:
I think containsKey() and get() is duplicated. get() can cover containsKey(); Maybe should judge eventPublish != null?

Reply:
OK